sphinx_gallery/gen_rst.py

```diff
@@ -581,6 +581,9 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
                     example_rst += code_output
                 else:
                     example_rst += code_output
+                    if not 'sphx-glr-script-out' in code_output:
+                        # Add some vertical space after output
+                        example_rst += "\n\n|\n\n"
                     example_rst += codestr2rst(bcontent) + '\n'
```

Comment:
In my opinion it would be convenient to have the space always and skip the check. I was trying to get it with CCS but is just too complicated and not done.
I would put all here
```python 
example_rst += '|\n\n' + codestr2rst(bcontent) + '\n'
```